components/brave_rewards/browser/rewards_notification_service_impl.h

```diff
@@ -9,11 +9,14 @@
 #include <string>
 
 #include "brave/components/brave_rewards/browser/rewards_notification_service.h"
+#include "extensions/buildflags/buildflags.h"
 
 class Profile;
 
 namespace brave_rewards {
 
+class ExtensionRewardsNotificationServiceObserver;
```

Comment:
I think this should be ok, @SergeyZhukovsky if you have any problems with it just fix after